chef/cookbooks/network/templates/default/ovs-pre-up.sh.erb

```diff
@@ -12,3 +12,17 @@ ovs-vsctl del-fail-mode <%= @bridgename %>
 <%
   end
 -%>
+
+# Open vSwitch older than 2.11 is starting 2*$(nproc)*3/4 netlink sockets per
+# number of cpu cores and per port. On machines with high core
+# count (e.g. 56 or higher) this can quickly exceed the total fd
+# limit set by ovs on itself on startup of 65536.
+
+# By Limit the handler threads to 8 per port we can handle ~ 4000 ports
+# instead of just ~ 700.
+# see https://mail.openvswitch.org/pipermail/ovs-dev/2018-September/352402.html
+# see bsc#1110865
+max_handler_threads=8
```

Comment:
Should we really hard code this here? I guess we'd better make this a chef attribute (not a proposal attribute just default attribute defined in the recipe that can be overridden node the node level)  and pass via the template.